sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -24,31 +23,22 @@ export default abstract class HttpBaseClient {
     return HttpBaseClient.createAuthHeader(AuthService.shared.keycloak?.token);
   }
 
-  protected handleRequestError(e: any) {
-    console.log(e);
-  }
-
-  protected async getCall(url: string, authHeader?: any): Promise<any> {
-    try {
-      const response = await this.apiClient.get(
-        url,
-        authHeader ?? this.addAuthHeader(),
-      );
-      return response;
-    } catch (error) {
-      this.handleRequestError(error);
-      throw error;
-    }
+  protected handleRequestError(error: unknown) {
+    console.log(error);
+    this.handleAPICustomError(error);
   }
 
-  protected async getCallTyped<T>(url: string, authHeader?: any): Promise<T> {
+  protected async getCall<T>(
```

Comment:
There is just one method that needs to execute this without the header. Maybe we can have a `unauthenticateGet` method to handle that special case and simplify the `getCall`.

Reply:
ya, for that i add the 3rd param `noHeader = false,`

Reply:
![image](https://user-images.githubusercontent.com/77353155/212405022-f8ff7815-4984-482d-927c-ce720a7c3932.png)